recipes/centos_vertx/Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN yum -y update && \
     useradd -u 1000 -G users,wheel -d /home/user --shell /bin/bash -m user && \
     usermod -p "*" user
 
+RUN mkdir -p /projects && chmod -R 0755 /projects
```

Comment:
@ibuziuk I think that `chmod -R 0777 /projects` here would be better otherwise non root users won't be able to create new subfolders inside `/projects`